Makefile

```diff
@@ -75,9 +75,9 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) \
-		rbac:roleName=manager-role output:rbac:artifacts:config=config/rbac \
-		crd:generateEmbeddedObjectMeta=true output:crd:artifacts:config=config/crd/bases \
-		webhook output:webhook:artifacts:config=config/webhook \
+		rbac:roleName=manager-role output:rbac:artifacts:config=config/components/rbac \
```

Comment:
qq, @Edwinhr716  why do we need to move these files to /components/

Reply:
good question, was following what was done by Kueue https://github.com/kubernetes-sigs/kueue and JobSet https://github.com/kubernetes-sigs/jobset, but turns out the whole reorganization isn't needed for it to work. Fixed it